lib/puppet/functions/simplib/to_integer.rb

```diff
@@ -0,0 +1,21 @@
+# Converts the argument into an `Integer`.
+# Only works if the passed argument responds to the `to_i()` Ruby method.
+#
+Puppet::Functions.create_function(:'simplib::to_integer') do
```

Comment:
@jeefberkey We don't want this to fail, we want it to convert something to an integer I don't think that Puppet can cast variables.